app/models/user.rb

```diff
@@ -6,6 +6,7 @@ class UserDisabled < StandardError
 
 class User < ActiveRecord::Base
   rolify
+  has_many :physical_tickets, through: :ticket_purchases, dependent: :destroy
```

Comment:
But I already have a through relation for conference in the physical ticket and still it didn't work. :confused: